lib/processors/jsdoc/lib/ui5/plugin.js

```diff
@@ -646,6 +835,11 @@ function convertValueWithRaw(node, type, propertyName) {
 				raw: local.raw
 			};
 		}
+
+		const potentialEnum = resolvePotemtialEnum(node, type);
```

Comment:
This is the case where the ENUM could be just the element. For example, `const { TitleLevel } = sap.ui.core;`